test/unittests/state_bloom_test.cpp

```diff
@@ -0,0 +1,64 @@
+// evmone: Fast Ethereum Virtual Machine implementation
+// Copyright 2023 The evmone Authors.
+// SPDX-License-Identifier: Apache-2.0
+
+#include "../utils/utils.hpp"
+#include <gtest/gtest.h>
+#include <test/state/bloom.hpp>
+#include <test/statetest/statetest.hpp>
+
+TEST(state_bloom, combine_blooms)
+{
+    using namespace evmone::state;
+
+    // https://sepolia.etherscan.io/block/2946594
+
+    const auto block_bloom = from_hexx(
```

Comment:
Instead of `from_hexx()` you can use `""_hex` literal. It also omits whitespace.